posts/d3-tutorial-points-axes.md

```diff
@@ -0,0 +1,139 @@
+   1. If you don't have http-server installed, retry the above after running the command: `npm install http-server -g`
+
+You should now be able to navigate to the example UI by navigating to http://localhost:8888/src/alignment-axis.html
+
+## Features
+
+The code takes a linear scale from 0-4, maps those values to categories from the popular tabletop RPG "Dungeons & Dragons", then allows you to quickly plot a point on those axes by entering an X and Y value bounded by \[0,4\]. Additionally, the points plotted will be color scaled from Light Blue (Lawful / Good) to Dark Red (Chaotic / Evil). This is to make it a little prettier than using a standard black circle, as well as provide more visual feedback. Remember: visualization is about communicating data-- the more communication you have, the better!
+
+## The Code
+
+## Variables and scales.
+
+           width = 800,
+           margin = 150,
+           min = 0,
+           max = 4,
+           lawfulness = ["Lawful", "Orderly", "Neutral", "Whimsical", "Chaotic"],
+           goodness = ["Good", "Wholesome", "Neutral", "Naughty", "Evil"];
+
+Height, Width, and Margin affect the size and spacing around the entirety of the graph.
+
+Min and Max represent the domain of the graph-- what value do we start and end at? Since we have 5 items in our array, the scale would be from 0-4, in this case.
+
+Lawfulness & goodness represent the two axis that D&D Characters are "aligned" to-- but in our example, it is the labels we want to use for our axis, basically.
+
+     var svg = d3
+           .select("body")
+           .append("svg")
+           .attr("class", "axis")
+           .attr("width", width)
+           .attr("height", height);
+
+The above code simply selects the "body" element of the HTML page, and adds an "SVG" element to it, defining a class of axis, and giving it a width and height of the appropriate values. This is what will contain our two axis.
+
+           xAxisLength = width - 2 * margin;
+
+`yAxisLength` and `xAxisLength` determine the size of those axes, which is scaled to the height & width of the SVG item we're working within, respectively, while also taking into account the margin.
+
+    var xScale = d3
+             .scaleLinear()
+             .domain([0, lawfulness.length])
+             .range([0, xAxisLength]),
+           yScale = d3
+             .scaleLinear()
+             .domain([0, goodness.length])
+             .range([0, yAxisLength]),
+
+Here's where things start to get a bit fun! xScale and yScale create two linear scales from 0 to 4 (our max/min sizes), and set the Range to be from 0 to the length of that axis. Think of the range as a ruler, fixed in size, while "Domain" is the values on that ruler. For example, if I was to label each third inch on a ruler with it's own label (such as "A", "B", "C", "D"), then the domain would be 0-12 inches while the Range would be \["A", "B", "C", "D"\].
```

Comment:
xScale and yScale could be wrapped like `xScale` and `yScale` since they are variables

Reply:
Also, I don't think [] characters need to be escaped

Reply:
the escaping happened automatically via the editor. *shrug* 

I dont know how to "wrap" it like that.